### PR TITLE
WIP <change yaml file to verify signning of NuGet.exe & Mssign.exe, 1st test>

### PR DESCRIPTION
**Pull request by heng-liu**
## Bug

Fixes: [#7668](https://github.com/NuGet/Home/issues/7668)
Regression: No  
* Last working version:   
* How are we preventing it in future:   

## Fix

Details: add verifying task in  vsts_build.yaml file

## Testing/Validation

Tests Added: No  
Reason for not adding tests:  
Validation:

### CODE REVIEW

File: build/vsts_build.yaml [hunk 1]
```diff
@@ -224,6 +224,25 @@ phases:
     inputs:
       TargetFolder: '$(Build.Repository.LocalPath)\\artifacts\\$(NupkgOutputDir)'
 
+  - task: PowerShell@1
+    displayName: "test only : check curr folder to see if there is nuget.exe file"
+    inputs:
+      scriptType: "inlineScript"
+      inlineScript: |
+        pwd
+        cd '$(Build.Repository.LocalPath)\\artifacts'
```
Comment (nkolev92) [inline build/vsts_build.yaml:233]: Consider using $(VsixPublishDir)

If you look at the output from Verify Assembly Signatures and StrongName you will see that the NuGet.exe in the nupkg is actually verified, at least the one from the nupgk. 
The nuget.exe in the nupkg should be same as the one in the artifacts folder. 

A simple solution might be just to package the mssign.exe

Comment (heng-liu) [reply build/vsts_build.yaml:233]: Thanks for telling about $(VsixPublishDir) 
I have a question, are we going to verify .exe directly or package the .exe then verify?

Comment (nkolev92) [reply build/vsts_build.yaml:233]: That's up to you, whatever is easier. 
If you can verify the exe directly, that's perfect.

**Comment by nkolev92** [comment]
The PR https://github.com/NuGet/NuGet.Client/pull/2798 contains validation for nuget.exe, nuget.mssign.exe and more. 

Closing this PR in favor of that one.

**Pull request closed without merging**